lnbits/templates/base.html

```diff
@@ -258,6 +258,62 @@
         { value: 'fi', label: 'Suomi', display: '🇫🇮 FI' }
       ]
     </script>
+    <script>
+      const currentUrl = window.location.href
+      let url = new URL(currentUrl)
+      let params = new URLSearchParams(url.search)
+      let q = Quasar.plugins.LocalStorage
+
+      const setColors = () => {
```

Comment:
why define a separate internal function if only called once?

Reply:
isnt it neater?

Reply:
it was already set this way... i can change it, but looks neater as @arcbtc said!

does it have memory/performance impact?